settings.gradle

```diff
@@ -27,7 +27,7 @@ develocity {
     projectId = "kafka"
     buildScan {
         uploadInBackground = !isCI
-        publishing.onlyIf { it.authenticated }
+        publishing.onlyIf { false }  // Still allows users/CI to run with --scan
```

Comment:
This is unrelated to this PR.  How about CI run with `--scan` only if `secrets.GE_ACCESS_TOKEN` is existent? That can avoid the false error `Publishing failed.` in normal PR.

Reply:
I'll remove this change from this PR. I added because I noticed my local builds were all publishing scans after I authenticated with ge.apache.org.